indico/modules/events/registration/client/js/form/fields/TimetableSessions.jsx

```diff
@@ -0,0 +1,199 @@
+function TimetableSessionsComponent({htmlId, htmlName, disabled, title, isRequired, ...props}) {
+  const [sessionData, setSessionData] = useState([]);
+  const {eventId} = useSelector(getStaticData);
+
+  useEffect(() => {
+    const fetchSessionData = async () => {
+      try {
+        const sessionsResponse = await indicoAxios.get(timeTableURL({event_id: eventId}));
+        setSessionData(sessionsResponse.data);
+      } catch (e) {
+        handleAxiosError(e);
+        return;
+      }
+    };
+    fetchSessionData(eventId);
+  }, [eventId]);
+
+  return (
+    <Form.Group styleName="accompanyingpersons-field" name={htmlName} id={htmlId}>
+      {sessionData.map((day, index) => (
+        <SessionBlock
+          key={index}
+          sessionBlock={day}
+          isRequired={isRequired}
+          disabled={disabled}
+          {...props}
+        />
+      ))}
+    </Form.Group>
+  );
+}
+
+TimetableSessionsComponent.propTypes = {
+  htmlId: PropTypes.number,
+  htmlName: PropTypes.string,
+  disable: PropTypes.bool,
+  title: PropTypes.string,
+  isRequired: PropTypes.bool,
+  display: PropTypes.string,
+};
+
+TimetableSessionsComponent.defaultProps = {};
+
+function transformDate(YYYYMMDD) {
+  const year = YYYYMMDD.slice(0, 4);
+  const month = YYYYMMDD.slice(4, 6) - 1;
+  const day = YYYYMMDD.slice(6, 8);
+
+  const date = new Date(year, month, day);
+
+  return date.toDateString();
+}
+
+function SessionBlock({sessionBlock, isRequired, disabled, ...props}) {
+  const sessionKey = Object.keys(sessionBlock)[0];
+  const sessionBlocks = Object.values(sessionBlock[sessionKey]).filter(
+    b => b.entryType === 'Session'
+  );
+  const formatDate = transformDate(sessionKey);
+
+  const checkboxName = `${name}-checkbox`;
+  const expanded = props.display === 'collapsed';
+
+  return (
+    <>
+      {sessionBlocks.length > 0 ? (
+        <div className="accordion" data-expanded={expanded} role="tablist" key={sessionKey}>
+          <h3 tabIndex={0} className="ui-accordion-header ui-state-default ui-accordion-icons">
+            <FinalCheckbox label={formatDate} name={checkboxName} key={formatDate} />
+          </h3>
+        </div>
+      ) : (
+        <></>
+      )}
+      {sessionBlocks.map(value => (
+        <FinalCheckbox label={value.title} name={value.title} key={value.title} />
+      ))}
+    </>
+  );
+}
+
+SessionBlock.propTypes = {
+  sessionBlock: PropTypes.object.isRequired,
+  isRequired: PropTypes.bool,
+  display: PropTypes.string,
+  minimum: PropTypes.number,
+  maximum: PropTypes.number,
+  disabled: PropTypes.bool,
+};
+
+SessionBlock.defaultProps = {
+  isRequired: false,
+  display: 'expanded',
+  minimum: 0,
+  maximum: 0,
+  disabled: false,
+};
+
+export default function TimetableSessionsInput({
+  htmlId,
+  htmlName,
+  disabled,
+  title,
+  isRequired,
+  ...props
+}) {
+  return (
+    <FinalField
+      id={htmlId}
+      name={htmlName}
+      component={TimetableSessionsComponent}
+      {...props}
```

Comment:
This is causing the duplicate description - do not blindly forward any props you have access to to the underlying field, but only those you actually need. Passing a description to a `FinalField` causes it to display it.

Reply:
That's one of the first things I thought, but as you can see, the props now doesn't include the description, that's why i discarded this possibility
![Uploading Screenshot 2024-02-15 at 12.27.28.png…]()

Reply:
Yeah, that's why I deleted the message, realised later I was looking at the wrong component, tyvm, bug solved